tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -782,6 +782,7 @@ pfcwd/test_pfcwd_warm_reboot.py:
      conditions:
         - "'dualtor' in topo_name"
         - https://github.com/sonic-net/sonic-mgmt/issues/8400
+        - "hwsku in ['Arista-7050CX3-32S-C32', 'Arista-7050CX3-32S-D48C8'] and release in ['202305', 'master']"
```

Comment:
@Gfrom2016 in our nightly test, I found release is  "release": "unknown" in fact.
Could you please check if it should be 202305 or 20230531?

Reply:
This has been fix in 20230531.03, release is '202305'